src/test/java/plugins/MatrixAuthPluginTest.java

```diff
@@ -52,22 +53,24 @@ public void helloWorld() throws Exception {
 
         j.save();
 
+        jenkins.logout();
+
         // if we login as Bob, he shouldn't see the job
         jenkins.login().doLogin("bob");
 
-        // wait for main panel to appear to make sure page is rendered
-        waitFor(by.id("main-panel"), 10);
+        // check for job's existence
+        driver.get(jenkins.getCurrentUrl() + "job/" + j.name + "/");
```

Comment:
@raul-arabaolaza Have you found something to get the Jenkins URL? `jenkins.getCurrentUrl()` may not be sure.